src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -31,6 +33,12 @@
     @JoinColumn(name = "group_id", nullable = false)
     private Group group;
 
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(name = "join_product_order",

```

Comment:
jak mamy tabelkę `products` i `orders` to najczęściej chyba tabelkę łączącą nazywa się `product_orders` lub `products_orders`

Reply:
Zrobione było tak jak w module. Zmieniłem na "products_orders". :-)